scripts/cypress.js

```diff
@@ -78,6 +63,8 @@ if (argv.mode === 'open') {
       if (result.totalFailed) {
         throw new Error(`${result.totalFailed} failing E2E tests`);
       }
+
+      process.exit(0);
```

Comment:
Since there is an extra process started by cypress (dev server), sometimes it can fail to terminate. Added `process.exit` to make sure the process exits

Reply:
it is not I see, ty